src/app/ubs/ubs-admin/components/ubs-admin-order/ubs-admin-order.component.ts

```diff
@@ -355,6 +355,11 @@ export class UbsAdminOrderComponent implements OnInit, OnDestroy, AfterContentCh
 
     if (changedValues.exportDetailsDto) {
       this.formatExporteValue(changedValues.exportDetailsDto);
+    } else {
+      const exportDetail1s = this.orderForm.get('exportDetailsDto').value;
+
+      changedValues.exportDetailsDto = exportDetail1s;
```

Comment:
remove exportDetails const
changedValues.exportDetailsDto = this.orderForm.get('exportDetailsDto').value;